pkg/entities/types.go

```diff
@@ -16023,6 +16040,11 @@ type entitySearchResponse struct {
 // AgentApplicationSettingsErrorCollectorHttpStatus - A list of HTTP status codes and/or status code ranges, such as "404" or "500-599"
 type AgentApplicationSettingsErrorCollectorHttpStatus string
 
+// AgentApplicationSettingsRawJsConfiguration - The "raw" configuration values for configuring the javascript client.
+// NOTE: Datatype "string" generated by Tutone manually changed to "map[string]interface{}" as this is a JSON, but fetched
+// as a string as the equivalent attribute on NerdGraph `jsConfig` is a string and has no nested attributes. Kindly DO NOT modify.
+type AgentApplicationSettingsRawJsConfiguration map[string]interface{}
```

Comment:
Thanks for adding this comment. This will help future devs know this was manually added then they see the diff. 

That being said, I _think_ you should be able to override its type generation in the `.tutone.yml` config file by adding the following to the entities package[ `types`](https://github.com/newrelic/newrelic-client-go/blob/main/.tutone.yml#L647) config:
```yaml
- name: AgentApplicationSettingsRawJsConfiguration
  create_as: "map[string]interface{}"
```

Reply:
Thank you, addressed this in the most recent commit. (https://github.com/newrelic/newrelic-client-go/pull/1035/files#r1356049860)